cli/wallet/wallet.go

```diff
@@ -164,10 +164,6 @@ func NewCommands() []cli.Command {
 						Name:  "name, n",
 						Usage: "Optional account name",
 					},
-					cli.StringFlag{
-						Name:  "contract",
-						Usage: "Verification script for custom contracts",
-					},
```

Comment:
This for non-standard non-deployed contracts, it has to be supported.

Reply:
Removed this commit. Just a question: why do we need this, if we can only work with standard contracts signatures?

Reply:
Verification scripts can contain any NeoVM code and we need to support it. Maybe we'll extend this support in future, but for now at least we need to be able to import these scripts into wallet.